openwisp_utils/admin_theme/templates/openwisp_utils/email_template.html

```diff
@@ -84,30 +83,45 @@
 
     .call-to-action {
       text-align: center;
-      margin-top: 1.88rem;
+      margin-top: 1.5rem;
     }
 
     .email-title {
       font-weight: 600;
       font-size: 1.13rem;
-      width: fit-content;
-      margin: auto;
-      margin-top: 1rem;
+      text-align: center;
+      line-height: 1.75rem;
       color: #101828;
     }
 
     .subtitle {
       font-weight: 400;
       font-size: 0.88rem;
-      width: fit-content;
-      margin: 0 auto 0.88em auto;
+      line-height: 1.25rem;
+      text-align: center;
+      margin-top: 0.1rem;
       color: #475467;
     }
 
+    hr {
```

Comment:
Gmail doesn't seem to want to use this color. It's a lot darker, especially on desktop:

<img width="703" height="531" alt="Screenshot from 2025-08-11 20-31-22" src="https://github.com/user-attachments/assets/1ec9fe30-577f-41e6-a055-a2916729b4e4" />

Can you try using any other element for which we know email clients will respect colors? What about an empty div?
Eg: `<div class="hr"></div>`.